### PR TITLE
docs: Add supported OCP version table and BIOS config table

### PR DESCRIPTION
**Pull request by hershpa**
Signed-off-by: Hersh Pathak <hersh.pathak@intel.com>

### CODE REVIEW

**Comment by vbedida79** [comment]
lgtm

**Comment by hershpa** [comment]
Thanks for reviewing. I made sure to use RHOCP instead of OCP to align with all our other readme documentation,

File: docs/supported_platforms.md [hunk 1]
```diff
@@ -30,6 +30,24 @@ The following Intel feature technologies are supported in the 1.0.1 release.
 | Intel® Data Center GPU Flex Series            | [Intel Data Center GPU Flex 140](https://www.intel.com/content/www/us/en/products/sku/230020/intel-data-center-gpu-flex-140/specifications.html) <br/>  [Intel Data Center GPU Flex 170](https://www.intel.com/content/www/us/en/products/sku/230019/intel-data-center-gpu-flex-170/specifications.html)  |
 | Intel® QuickAssist Technology (Intel® QAT) | [4th Gen Intel® Xeon® Scalable processors](https://www.intel.com/content/www/us/en/products/docs/processors/xeon-accelerated/4th-gen-xeon-scalable-processors.html) |                               
 
+## Setting up Intel Hardware Features
+
+### BIOS Configuration
+Note: Please refer to your BIOS vendor for specific instructions. This is only a reference for BIOS configuration. 
+| Feature | BIOS Configuration | 
+| ----- | ---------------------- |
+| SGX | [Link](https://www.intel.com/content/www/us/en/support/articles/000087972/server-products/single-node-servers.html) |
```
Comment (chaitanya1731) [inline docs/supported_platforms.md:39]: Can you keep the 'Feature' column consistent in both tables.. You can keep it similar to Supported RHOCP versions table's feature column

**Review by chaitanya1731 — CHANGES_REQUESTED** [review]

File: docs/supported_platforms.md (hunk 1 above)
Comment (hershpa) [reply docs/supported_platforms.md]: Good catch! Thanks for the review. Done, updated PR!

**Review by vbedida79 — APPROVED** [review]

**Review by chaitanya1731 — APPROVED** [review]

**Comment by uMartinXu** [comment]
It looks good to me!

**Pull request merged**